whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageService.java

```diff
@@ -58,24 +59,26 @@ public EmailMessageInfo getAutomatedFailureMessageInfo(final MimeMessage message
         return autoSubmittedMessageParser.parse(message);
     }
 
-    public void verifyAndSetAsUndeliverable(final EmailMessageInfo message) {
-        final List<String> outgoingEmail = outgoingMessageDao.getEmails(message.messageId());
+    public void verifyAndSetAsUndeliverable(final EmailMessageInfo messageInfo) throws MailParsingException {
```

Comment:
Where does a `MailParsingException` end up? Make sure it doesn't break the Mailupdate message flow (i.e. the message should still be handled cleanly)

Reply:
at this point the message has been detected as bounced message and this error occurs because it cannot be transform into bytes[]
Do we still want to continue with the flow? in this situation should I attempt just storing the bounced email without storing the byte[]?

Reply:
I pushed doing this, in case of this issue the message will not be stored, just the email and the status (as we are doing now)

Reply:
MailParsingException is now handled properly and not propogated.